README.md

```diff
@@ -1,10 +1,9 @@
 # Filecoin Devgrants
+> We are continously reviewing applictions for [open grants](https://github.com/filecoin-project/devgrants/blob/master/open-grant-proposals/open-proposal-template.md), [RFPS](https://github.com/filecoin-project/devgrants/tree/master/rfps), and [microgrants](https://github.com/filecoin-project/devgrants/blob/master/microgrants/microgrants.md). Submit your proposals now for priority consideration.
 
-> **Wave 8 dev grant proposals are due June 30, 2021 for priority consideration. We will continue to evaluate new proposals after that deadline as capacity allows.**
+We are also accepting RFPs built on Hedera and Filecoin. Read more about this grant category [here](https://github.com/filecoin-project/devgrants/blob/hedera-filecoin-joint-grant/rfps/hedera-and-filecoin.md). 
```

Comment:
```suggestion
```
This is not live yet (details will be finalized in a call later this week) so let's remove for now.